.github/workflows/checks.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: dart-lang/setup-dart@v1
         with:
-          sdk: stable
+          sdk: 2.18.7
```

Comment:
We currently only support dart `2.x`, will have to spend some time considering how to handle multiple versions of dart

Reply:
Yeah that's interesting. I've got some ideas if you want to chat about it

Reply:
Definitely, I have one idea, but it doesn't really improve much, would love to chat